src/App.js

```diff
@@ -10,6 +10,7 @@ import { ratingFilter } from "./utils/productFilters/rating";
 import { priceRangeFilter } from "./utils/productFilters/priceRangeFilter";
 import { fastDeliveryFilter } from "./utils/productFilters/fastDeliveryFilter";
 import { outOfStockFilter } from "./utils/productFilters/outOfStockFilter";
+import { searchFilter } from "./utils/productFilters/searchFilter";
```

Comment:
you can use barrel import, it will reduce the lines of code

Reply:
done all the changes. Thank you